.travis.yml

```diff
@@ -1,27 +1,18 @@
 language: node_js
 node_js:
-  - "node"
+- node
 notifications:
   email: false
-addons:
-  apt:
-    packages:
-    - libgconf-2-4
 cache:
-  npm: true
   directories:
-    - ~/.npm
-    - ~/.cache
+  - "~/.npm"
+  - "~/.cache"
   override:
-    - npm ci
-    - npm run cy:verify
-install:
   - npm ci
+  - npm run cy:verify
+before_script:
+  - npm start --silent &
 script:
-  - npm start &
   - npm run start:ci &
   - cypress run --record --key 86ce623b-9bbc-4993-8dca-f3535c528333
-deploy:
-  provider: vercel
-  app: curatour
-  skip_cleanup: true
```

Comment:
I think this might have been the main issue.

Reply:
Thanks for working through that!